src/js/options.js

```diff
@@ -16,25 +16,24 @@ async function requestPermissions() {
 
 async function enableDisableSync() {
   const checkbox = document.querySelector("#syncCheck");
-  if (checkbox.checked) {
-    await browser.storage.local.set({syncEnabled: true});
-  } else {
-    await browser.storage.local.set({syncEnabled: false});
-  }
+  await browser.storage.local.set({syncEnabled: !!checkbox.checked});
   browser.runtime.sendMessage({ method: "resetSync" });
 }
 
+async function enableDisableReplaceTab() {
```

Comment:
funky way to name this, but I see it matches `enableDisableSync` so better to be consistently funky than pedantically un-funky.

Reply:
Exactly what I thought. Maybe refactoring to `toggleSync` and `toggleReplaceTab` would be a good idea but I wasn't sure enough.